tools/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
-add_executable(
-	HogMaker
-	HogMaker/HogFormat.cpp
-	HogMaker/HogMaker.cpp
-)
-add_dependencies(HogMaker get_git_hash)
-target_include_directories(HogMaker PRIVATE ${PROJECT_BINARY_DIR}/lib)
+if (ANDROID)
+	find_package(HogMaker REQUIRED)
```

Comment:
redundant with #465

Reply:
this PR has a bunch of those other PRs merged into it :)